docs/src/modules/components/MarkdownElement.js

```diff
@@ -279,10 +279,52 @@ const Root = styled('div')(({ theme }) => ({
       marginTop: theme.spacing(1),
     },
   },
+  '& .MuiCode-root': {
+    position: 'relative',
+    '&:hover': {
+      '& .MuiCode-copy': {
+        opacity: 1,
+      },
+    },
+  },
+  '& .MuiCode-copy': {
+    display: 'none',
+    opacity: 0,
+    backgroundColor: alpha(blue[800], 0.3),
+    cursor: 'pointer',
+    position: 'absolute',
+    top: theme.spacing(1),
+    right: theme.spacing(1),
+    fontFamily: 'inherit',
+    fontSize: '0.75rem',
+    padding: theme.spacing(0.5, 1),
+    borderRadius: 4,
+    border: `1px solid`,
+    borderColor: blueDark[500],
+    color: `rgba(255,255,255,0.8)`,
+    '&:hover, &:focus': {
+      opacity: 1,
+      color: '#fff',
+    },
+    '&[data-copied]': {
+      // style of the button when it is in copied state.
+      borderColor: blue[400],
+      color: '#fff',
+      backgroundColor: blueDark[600],
+    },
+    '&:focus-visible': {
```

Comment:
If we want to be strict, then we would need to use ButtonBase or our hook
```suggestion
    '&.Mui-focusVisible': {
```

https://caniuse.com/?search=focus-visible vs. https://mui.com/material-ui/getting-started/supported-platforms/

Reply:
In our case, it is tricky. There are 2 places:
1. the button generated at file system read (markdown file)
2. the button created by React

It is possible to use ButonBase for the second one but I don't think the first point is possible, that's why I decide to go with `focus-visible` to keep the consistency. For a safer choice, we could go with `:focus`.